### PR TITLE
[New] Object Storage Migration guides

### PR DESCRIPTION
**Pull request by jddocs**
Three new guides for migrating third-party object storage data to Linode Object Storage:
- Migrate From Azure Blob Storage to Linode Object Storage
- Migrate From AWS S3 to Linode Object Storage
- Migrate From Google Cloud Storage to Linode Object Storage

### CODE REVIEW

**Comment by jddocs** [comment]
Initial commit with Azure guide. Other two guides not included yet & dictionary not updated.

File: docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+
+    ![Rclone-global-stats](Rclone-global-stats.jpg)
+
+    ![Rclone-speed](Rclone-speed.jpg)
+
+1.  The WebUI will disconnect automatically when the copy job is complete.
+
+## Verify the Migration
+
+You can compare the number of objects in both your source and target buckets along with the total size of the buckets to verify full completion of the copy job.
```
Comment (nmelehan) [inline docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md:256]: I'm wondering if there's some kind of checksum/hash comparison that could be done that also verifies the contents of the buckets. There might not be, but it was just a thought that came to me

File: docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+title: "How to Migrate From AWS S3 to Linode Object Storage"
+description: "This guide includes steps for how to migrate content from AWS S3 to Linode Object Storage using rclone."
+authors: ["John Dutton"]
+contributors: ["John Dutton"]
+published: 2024-09-23
+keywords: ['migrate','migration','object storage','aws','s3','rclone']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+external_resources:
+- '[Linode Object Storage product documentation](https://techdocs.akamai.com/cloud-computing/docs/object-storage)'
+- '[Linode Object Storage guides & tutorials](/docs/guides/platform/object-storage/)'
```
Comment (nmelehan) [inline docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md:12]: should we add an alias from the old /guides/migrate-to-linode-object-storage/ guide to this new S3 one?

**Review by nmelehan — APPROVED** [review]

File: docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+    -   {{< placeholder "AWS-SECRET-KEY" >}}: Your AWS secret key
+    -   {{< placeholder "us-east-1" >}}: The AWS region ID for your S3 bucket
+
+    **Linode Object Storage**
+    -   {{< placeholder "LINODE-ACCESS-KEY" >}}: Your Linode Object Storage access key
+    -   {{< placeholder "LINODE-SECRET-KEY" >}}: Your Linode Object Storage secret key
+    -   {{< placeholder "us-lax-1" >}}: The region ID for your Linode Object Storage bucket
+
+    ```file
+    [aws-perf-test-source]
```
[the remark this answers is not in the document]
Comment (jddocs) [reply docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md:140]: good catch - edited

File: docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md (hunk 1 above)
Comment (jddocs) [reply docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md:256]: will check on this with my SME for a future edit

File: docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+
+![S3-to-OBJ-Arch2](S3-to-OBJ-Arch2.png)
+
+## Prerequisites and Required Information
+
+-   **A virtual machine with rclone installed**. This guide recommends a 16GB dedicated virtual machine with 8 CPU cores. The plan you require may vary depending on your workload.
+
+-   The **public IPv4 address** of your virtual machine.
+
+-   As a security best practice, **use a firewall to only allow inbound port 5572**. This is the default port used by rclone and enables secure access to the WebUI since it is served over HTTP.
```
[the remark this answers is not in the document]
Comment (jddocs) [reply docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md:81]: added a line to address this and will link to resources at a later date

**Pull request merged**